spec.html

```diff
@@ -41522,6 +40971,7 @@ <h1>Properties of the AsyncFunction Prototype Object</h1>
         <li>is <dfn>%AsyncFunction.prototype%</dfn>.</li>
```

Comment:
The line immediately before this one (which github won't let me address directly, gah) has a reference to `%AsyncFunctionPrototype%` and should be deleted.